### DISCUSSION TITLE
Should i buy [214]  EMOS E6016 weatherstation

### DISCUSSION THREAD

**Discussion by MatejsLab** [merbanan/rtl_433#2842] [Q&A]
I am searching for wind speed sensor for greenhouse automation, and wind alerts.

Does anyone have EMOS E6016 weatherstation, is it any good, does it read WindSpeed properlly in RTL433 ?

How well batteries last, is reception reliable and the range/distance for Blue (is better) or Black (clone) RTL reciver i have from Aliexpress?
This is the device i looked at.
https://www.emos-si.si/brezzicni-senzor-e06016

Are there any better for under 50€ for RTL433, or should i rather buy wired wind sensor from Aliexpress instead and connect it to ESP8266 Wifi micromontroller.

I am a fan of RTL433 for Temperaraute and rain sensors, that i have forwarded to MQTT and OpenHAB.
The only issue i had is that sensors change ID address on battery swap, i need to reconfigure them then.

Thanks for sharing your experience
Matej

**Comment by zuckschwerdt** [comment]
Some of the decoder authors and users are @stomko11 @AutomGuy @vidarino

**Comment by stomko11** [comment]
I sold mine. Ocasionally temperature went haywire and showed like 25°C (when it was 5 in reality). Also for wind, problem is reports every 30sec, which does not catch heavy wind gusts. And I did have some crazy wind spikes as well)
I ended up buying ecowitt. Shows wind gusts, has solar panel, lux sensor, and overall worked out of the box. In addition, rain measurements are more granular, shows event rain rate (aside from daily/weekly/total....). But it is not under 50€ (mine was around 100e).

**Comment by MatejsLab** [comment]
Oh thanks, it seemed suspicious how weel it works as it is cheap, so i
asked.
I appriciate your anwser so i dont buy potato station.

Your station looks really profesional.

I am considrring wired wind sensor then for reliability, i only car for
speed and response.

For rich data i might borow the "neighbours" tempest data online i just
checked that 3 new stations poped up locally in 5km range.
And they update quite fast.
So i may be able to pull data into OpenHab with scraping or API Binding.

The tempest seems topnotch if i use all 3 stations for reliability,
hopefully i can pull data every 10 to 30s.

Cheers
Matej






V ned., 18. feb. 2024, 17:51 je oseba stomko11 ***@***.***>
napisala:

**Comment by stomko11** [comment]
Reliability of sent data from Emos was alright. It is the data that emos sent what seemed to be off from time to time (ocassional spike of wind speed 140km/h).
Ecowitt allows setting reporting interval too. And it works anyway in a similar manner - rf between station and gateway, and gateway is connected to wifi. It has been rock solid since I bought it.
